lib/puppet/type/splunk_authentication.rb

```diff
@@ -1,24 +1,7 @@
+require 'puppet_x/puppetlabs/splunk/type'
```

Comment:
Anytime we do this, we should be doing `require File.join(File.dirname(__FILE__),.....etc.etc.`